src/Samples/DIPS.Xamarin.UI.Samples/Controls/Popup/PopupPage.xaml

```diff
@@ -52,30 +50,30 @@
                     BackgroundColor="DarkGray"
                     TextColor="WhiteSmoke">
                 <Button.Behaviors>
-                    <popup:PopupBehaviour PopupBindingContextFactory="{Binding Source={x:Reference btn2}, Path=BindingContext.FilterViewModelFactory2}"
+                    <popup:PopupBehavior BindingContextFactory="{Binding Source={x:Reference btn2}, Path=BindingContext.FilterViewModelFactory2}"
                                           Direction="Auto">
-                        <popup:PopupBehaviour.PopupContent>
+                        <popup:PopupBehavior.Content>
                             <Frame CornerRadius="30"
-                                   Margin="0,20,10,0"
-                                   BackgroundColor="Orange">
+                                        Margin="0,20,10,0"
+                                        BackgroundColor="Orange">
                                 <StackLayout Orientation="Vertical">
                                     <Label Text="Yolo Yolo Yolo Yolo Yolo Yolo Yolo" VerticalOptions="Center"/>
                                     <CheckBox IsChecked="{Binding Ascending}"
-                                              Margin="5"
-                                              HorizontalOptions="Center"
-                                              VerticalOptions="Center"/>
+                                                    Margin="5"
```

Comment:
Lets clean up sample page a little bit, less yolo and focus on readability. Keep examples short and simple.